charts/dbildungs-iam-server/values.yaml

```diff
@@ -1,18 +1,16 @@
-dbildungsIamExternalPort: 80
-dbildungsIamCpuMax: 2
-dbildungsIamMemMax: 4G
-dbildungsIamRedisCpuMax: 2
-dbildungsIamRedisMemMax: 4G
-dbildungsIamRedisCpuRequest: 200m
-dbildungsIamRedisMemRequest: 200Mi
-dbildungsIamReplications: 1
+dbildungsIamServerCpuMax: 2
```

Comment:
Please define resource requests and limits with usual pattern:
resources: 
  requests:
  limits:

Reply:
If possible I would only like to rename them here to get rid of the old expression. There are many things that need to get adjusted in this Helm Chart, I would like to do it in a follow up ticket. The primary focus on these PRs is to get the naming itself straight.